File: 30 day code c++/day6.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;

// this is a simple code to print the string
int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */   
    int n,i,j,t;
    cin>>t;
    for(i=0;i<t;i++)
    {
   string s;
   cin>>s;
   string s1,s2;
    for(j=0;j<s.size();j++)
    {
        if(j%2==0)
        {
            s1+=s[j];
        }
        else
        {
            s2+=s[j];
        }
    }    
    cout<<s1<<" "<<s2<<"\n";
    }
    return 0;
}

File: 30daysOFcode_Php/day6.php
<?php
$_fp = fopen("php://stdin", "r");
/* Enter your code here. Read input from STDIN. Print output to STDOUT */

$row = 0;
while($string = fgetcsv($_fp)){
    $str = $string[0];
    $even = $odd = '';
    $row++;
    if($row > 1){
        $i = 0;
        foreach(str_split($str) as $k => $v){
            if($i % 2 == 1){
                $odd .= $v;
            }else{
                $even .= $v;
            }
            $i++;
        }

        echo $even . ' '. $odd."\n";
    }
}

?>
